Library/Homebrew/sandbox.rb

```diff
@@ -162,6 +162,11 @@ class SandboxProfile
           (regex #"^/dev/ttys?[0-9]*$")
           )
       (deny file-write*) ; deny non-whitelist file write operations
+      (deny file-read*
```

Comment:
Probably want this in a `def deny_write_homebrew_something` method and then make it conditional on an environment variable being set.